webapp/locales/en.json

```diff
@@ -22,14 +22,14 @@
     "follow": "Follow",
     "followers": "Followers",
     "following": "Following",
-    "shouted": "Shouted",
+    "shouted": "Recommended",
     "commented": "Commented",
     "userAnonym": "Anonymous",
     "socialMedia": "Where else can I find"
   },
   "notifications": {
     "menu": {
-      "mentioned": "has mentioned you in a post"
+      "mentioned": "mentioned you in a post"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-07T19:13:11Z" title="Friday, June 7th 2019, 9:13:11 pm +02:00">Jun 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -14,22 +14,22 @@
     "hello": "Hello"
   },
   "editor": {
-    "placeholder": "Leave your inspirational thoughts..."
+    "placeholder": "Write something inspiring..."
   },
   "profile": {
     "name": "My Profile",
     "memberSince": "Member since",
     "follow": "Follow",
     "followers": "Followers",
     "following": "Following",
-    "shouted": "Shouted",
+    "shouted": "Recommended",
     "commented": "Commented",
     "userAnonym": "Anonymous",
     "socialMedia": "Where else can I find"
   },
   "notifications": {
     "menu": {
-      "mentioned": "has mentioned you in a post"
+      "mentioned": "mentioned you in a post"
```


It is better English to say `has mentioned` because it is not specific when the mention happened